content/dips/DIP-19.md

```diff
@@ -0,0 +1,40 @@
+---
+DIP: 19
+title: Move LP rewards formula weight from stkDYDX to MakerVolume
+status: Proposed
+author: Xenophon Labs (@XenophonLabs)
+shortDescription: Remove `stkDYDX` from LP rewards
+discussions: https://forums.dydx.community/discussion/7872-drc-remove-stkdydx-from-lp-rewards-formulas
+created: 2022-12-05
+---
+
+## Simple Summary
+
+Remove `stkDYDX` from the LP rewards mechanism, and reallocate the `stkDYDX` term's weight to `MakerVolume`.
+
+## Abstract
+
+We propose removing the `stkDYDX` term from the LP rewards mechanism and reallocating the `stkDYDX` term's weight to `MakerVolume`. The decision to remove the `stkDYDX` term follows naturally from the community's decision to wind down the safety staking module. The decision to reallocate the `stkDYDX` term's weight to the `MakerVolume` term comes from our opinion that volume is the best proxy for value that LPs provide to the protocol's traders.
+
+On December 10, 2022, there was a Snapshot vote on this which concluded with {numVoters} voters and {agreeingDYDX} DYDX in agreement ({agreementPercentage}) to make these LP rewards formula changes.
```

Comment:
Will need to fill in these placeholders before merging.

Reply:
Hey @maxholloway , can you include the numbers accordingly?
- {numVoters}:   883 
- {agreeingDYDX}:   16,177,534 
- {agreementPercentage}: 99.97%